UI/src/view/point_asset/trafficSignForm.js

```diff
@@ -56,6 +77,96 @@
         $('.form-traffic-sign select#trafficSigns_type').html(singleChoiceSubType(collection, $(event.currentTarget).val()));
         selectedAsset.setPropertyByPublicId('trafficSigns_type', $('.form-traffic-sign select#trafficSigns_type').val());
       });
+
+      rootElement.find('#additional-panel-checkbox').on('change', function (event) {
+        if(!$(event.currentTarget).prop('checked')) {
+          $('.panel-group-container').remove();
+          selectedAsset.setAdditionalPanels();
+        }
+        else {
+          $('.additional-panel-checkbox').after(renderAdditionalPanels({values:[defaultAdditionalPanelValue]}, collection));
+          setAllPanels();
+        }
+        bindPanelEvents();
+      });
+
+      rootElement.find('.form-traffic-sign-panel input[type=text]#panelValue, .form-traffic-sign-panel input[type=text]#panelInfo, .form-traffic-sign-panel select').on('change input', function (event) {
```

Comment:
can't be completely replaced, since the 'change' will be triggered when removing the panels with the checkbox. so  `'input[type=text]#panelValue, input[type=text]#panelInfo, .form-traffic-sign-panel select'` is a compromise